manifest.toml

```diff
@@ -54,9 +54,10 @@ ram.runtime = "50M"
 
 [resources]
         [resources.sources.main]
-        url = "https://files.spip.net/spip/archives/spip-v4.3.3.zip"
-        sha256 = "e2a7e6094a398843b83d22f6ab4ed2433ceea8278ef183da0e324c31ef95ac60"
-        in_subdir = false
+        url = "https://git.spip.net/spip/spip/-/archive/v4.3.5/spip-v4.3.5.tar.bz2"
```

Comment:
```suggestion
        url = "https://files.spip.net/spip/archives/spip-v4.3.5.zip"
```

Reply:
Thanks for your help, in fact we can only use `sha256` for the sources.

Reply:
The change to get tag assets from git was intentional, this way autoupdater can pull new tags whenever a new version comes out.
This was to speed up updates in YNH, but OTOH I see a number of stale update attempts so I'll leave it to whoever decides to maintain the app.
If however we go with 'official' source file `in_subdir` and `autoupdate.*` are to be scrapped probably.

Reply:
Well, the actual tag asset does not contain the whole source code of the app. 

@rodinux i can manage to provide a sha256 in the json i show you if needed

@orhtej2 we are in the process to provide a release arrtifact, but it'll take some weeks i guess